web/themes/custom/novel/css/novel.css

```diff
@@ -0,0 +1,4 @@
+.dpl-react-app-container--page {
+  min-height: 100vh;
+  background-color: #f6f5f0;
```

Comment:
Consider if this can be moved to the design system.

The information about the background color is directly related to the design of the entire application. It would be nice if we did not have to duplicate the color here.